rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -366,11 +371,11 @@ components:
     MetadataType:
       description: The type of metadata.
       type: string
-      enum: [irm]
+      example: irm
     Status:
       description: The status of the operation.
       type: string
-      enum: [In Progress, Done, Failed]
+      enum: [In Progress, Done, Failed, Unknown]
```

Comment:
❗️ We should not be adding the `Unknown` error at the API level. Unknown in our internal design should either translate in a 503 letting the client know they should retry or a redirect back to the public URL. Both situations are such that the request comes back into our endpoint and hits a newer version of the endpoint that can handle the updated data. The notion of unknown should never leave our bounded context.